vault/resource_consul_secret_backend_role.go

```diff
@@ -210,14 +247,22 @@ func consulSecretBackendRoleRead(d *schema.ResourceData, meta interface{}) error
 
 	// map request params to schema fields
 	params := map[string]string{
-		"policies":         "policies",
-		"max_ttl":          "max_ttl",
-		"ttl":              "ttl",
-		"token_type":       "token_type",
-		"local":            "local",
-		"consul_roles":     "consul_roles",
-		"consul_namespace": "consul_namespace",
-		"partition":        "partition",
+		"max_ttl":            "max_ttl",
+		"ttl":                "ttl",
+		"token_type":         "token_type",
+		"local":              "local",
+		"consul_roles":       "consul_roles",
+		"consul_namespace":   "consul_namespace",
+		"partition":          "partition",
+		"service_identities": "service_identities",
+		"node_identities":    "node_identities",
+	}
+
+	if _, ok := d.GetOk("policies"); ok {
+		params["policies"] = "policies"
+	}
+	if _, ok := d.GetOk("consul_policies"); ok {
+		params["consul_policies"] = "consul_policies"
```

Comment:
I think this be what you are after?

```suggestion
		"service_identities": "service_identities",
		"node_identities":    "node_identities",
		"policies":           "policies",
		"consul_policies":    "consul_policies",
```